cassandra-analytics-core/src/test/java/org/apache/cassandra/spark/validation/KeyStoreValidationTests.java

```diff
@@ -97,4 +98,15 @@ public void testValidKeyStore()
         Throwable throwable = validation.perform();
         assertNull(throwable);
     }
+
+    @Test
+    public void testExpiredKeyStore()
+    {
+        SecretsProvider secrets = TestSecretsProvider.forKeyStore("PKCS12", "keystore-expired.p12", "qwerty");
+        KeyStoreValidation validation = new KeyStoreValidation(secrets);
+
+        Throwable throwable = validation.perform();
+        assertInstanceOf(RuntimeException.class, throwable);
+        assertThat(throwable.getMessage()).startsWith("Certificate expired, valid NotAfter: ");
```

Comment:
the error message is odd, Should we just say:

`Certificate expired. NotAfter:` instead?

Reply:
Updated to `Certificate expired. ` instead